modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -17,45 +17,64 @@ and other settings.
 
 .Procedure
 
-. Create a `journald.conf` file with the required settings:
+. Create a Butane config file, `40-worker-custom-journald.bu`, that includes an `/etc/systemd/journald.conf` file with the required settings.
 +
-[source,terminal]
+[NOTE]
+====
+See "Creating machine configs with Butane" for information about Butane.
+====
++
+[source,yaml]
 ----
-Compress=yes <1>
-ForwardToConsole=no <2>
-ForwardToSyslog=no
-MaxRetentionSec=1month <3>
-RateLimitBurst=10000 <4>
-RateLimitIntervalSec=30s
-Storage=persistent <5>
-SyncIntervalSec=1s <6>
-SystemMaxUse=8g <7>
-SystemKeepFree=20% <8>
-SystemMaxFileSize=10M <9>
+variant: openshift
+version: 4.8.0
+metadata:
+  name: 40-worker-custom-journald
+  labels:
+    machineconfiguration.openshift.io/role: worker
+storage:
+  files:
+  - path: /etc/systemd/journald.conf
+    mode: 0644 <1>
+    overwrite: true
+    contents:
+      inline: |
+        Compress=yes <2>
+        ForwardToConsole=no <3>
+        ForwardToSyslog=no
+        MaxRetentionSec=1month <4>
+        RateLimitBurst=10000 <5>
+        RateLimitIntervalSec=30s
+        Storage=persistent <6>
+        SyncIntervalSec=1s <7>
+        SystemMaxUse=8g <8>
+        SystemKeepFree=20% <9>
+        SystemMaxFileSize=10M <10>
 ----
 +
-<1> Specify whether you want logs compressed before they are written to the file system.
+<1> Set the permissions for the `journal.conf` file. It is recommended to set `0644` permissions.
+<2> Specify whether you want logs compressed before they are written to the file system.
 Specify `yes` to compress the message or `no` to not compress. The default is `yes`.
-<2> Configure whether to forward log messages. Defaults to `no` for each. Specify:
+<3> Configure whether to forward log messages. Defaults to `no` for each. Specify:
 * `ForwardToConsole` to forward logs to the system console.
 * `ForwardToKsmg` to forward logs to the kernel log buffer.
 * `ForwardToSyslog` to forward to a syslog daemon.
 * `ForwardToWall` to forward messages as wall messages to all logged-in users.
-<3> Specify the maximum time to store journal entries. Enter a number to specify seconds. Or
+<4> Specify the maximum time to store journal entries. Enter a number to specify seconds. Or
 include a unit: "year", "month", "week", "day", "h" or "m". Enter `0` to disable. The default is `1month`.
-<4> Configure rate limiting. If, during the time interval defined by `RateLimitIntervalSec`, more logs than specified in `RateLimitBurst`
+<5> Configure rate limiting. If, during the time interval defined by `RateLimitIntervalSec`, more logs than specified in `RateLimitBurst`
 are received, all further messages within the interval are dropped until the interval is over. It is recommended to set
```

Comment:
```suggestion
<5> Configure rate limiting. If more logs are received than what is specified in `RateLimitBurst` during the time interval defined by `RateLimitIntervalSec`, all further messages within the interval are dropped until the interval is over. It is recommended to set
```
The sentence is kind of clunky, so suggesting a minor revision for clarity if the meaning is preserved.